models/RecruiterModel.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+const DataTypes = Schema.Types;
+
+const RecruiterSchema = new Schema(
+  {
+    companyName: {
+      type: DataTypes.String,
+      required: true,
+    },
+    companyLogo: {
+      type: DataTypes.ObjectId,
+      required: true,
+    },
+    companyPhotos: [DataTypes.ObjectId],
+    userRole: {
+      type: DataTypes.String,
+      required: true,
+    },
+    mobileNo: {
+      type: DataTypes.String,
+      required: true,
+    },
+    locations: [DataTypes.String],
+    website: {
+      type: DataTypes.String,
+      required: true,
+    },
+    foundationYear: {
+      type: DataTypes.Number,
+      required: true,
+    },
+    noOfEmployees: {
+      type: DataTypes.Number,
+      required: true,
+    },
+    aboutCompany: {
+      type: DataTypes.String,
+      required: true,
+    },
+    linkedinProfile: {
+      type: DataTypes.String,
+    },
+    twitterProfile: {
+      type: DataTypes.String,
+    },
+    facebookProfile: {
+      type: DataTypes.String,
+    },
+    notInterestedCandidates: {
+      type: [DataTypes.ObjectId],
+      default: [],
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 3 tabs but found 6 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
			default: [],
```